### PR TITLE
Plugin Generator Updates

### PR DESCRIPTION
**Pull request by Fryguy**
Plugin Generator Updates

### CODE REVIEW

**Comment by Fryguy** [comment]
@agrare Please review.

File: lib/tasks_private/yarn.rake [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace :yarn do
```
Comment (Fryguy) [inline lib/tasks_private/yarn.rake:1]: Is this still needed?  If so, should this be a common task pushed down to the provider generator?

Comment (agrare) [reply lib/tasks_private/yarn.rake:1]: The main ui-classic `update:yarn` already does `yarn install` for all plugins that bring js packages so I think we can remove this

**Review by agrare — APPROVED** [review]

File: lib/tasks_private/yarn.rake (hunk 1 above)
Comment (Fryguy) [reply lib/tasks_private/yarn.rake:1]: Ok, I'll have a follow up for these.

**Pull request merged**